conf/app.src

```diff
@@ -1,6 +1,7 @@
 SOURCE_URL=https://github.com/pluxml/PluXml/archive/v5.8.8.zip
```

Comment:
PluXml 5.8.8 zip seems to be unavailable unloke PluXml 5.8.9 that REVERT v5.8.8 FIX Static pages PHP injection vulnerability https://github.com/pluxml/PluXml/issues/558 (P3ter, Moritz Huppert)
SOURCE_URL=https://github.com/pluxml/PluXml/releases/download/5.8.9/pluxml-5.8.9.zip
e910d61041c56ce97085c0251f617ee2230ee7b7557e7af1c7cb513ce2d8b060

Reply:
also fixed, thanks